### DISCUSSION TITLE
Please Google play store ban/tarmenate problem. Or give us solution

### DISCUSSION THREAD

**Discussion by amjad019926** [Sketchware-Pro/Sketchware-Pro#879] [Q&A]
Is it true that making apps with sketchware pro will be Ban/tarmenate? Anyone go into detail? What can be done? plz help.
how to we can upload apps in playstore

**Comment by subhash16** [comment]
Your statement is false,If you make an app with sketchware pro there is no chance of ban/terminate console account.If you are making illigal app then you will terminated.

**Comment by j-px** [comment]
Google Play doesn't terminate apps just because they're made with Sketchware Pro. Your app of course has to follow its many guidelines and policies.
